scripts/examples/gn_efr32_example.sh

```diff
@@ -98,6 +98,12 @@ if [ "$#" == "0" ]; then
             Periodic query timeout variable for OTA in seconds
         rs91x_wpa3_only
             Support for WPA3 only mode on RS91x
+        sl_matter_version
+            Overwrite Matter software version
+        sl_matter_version_str
```

Comment:
There's a conflicting description of sl_matter_version_str at line 85 above

Reply:
removed duplicate description